tools/price_impact.php

```diff
@@ -0,0 +1,185 @@
+<?php
+require_once __DIR__ . '/inc/__tools_header.php';
+require_once DOL_DOCUMENT_ROOT.'/core/class/html.formfile.class.php';
+require_once DOL_DOCUMENT_ROOT.'/core/class/html.formcompany.class.php';
+require_once DOL_DOCUMENT_ROOT.'/core/class/html.formfile.class.php';
+require_once DOL_DOCUMENT_ROOT.'/core/class/html.formprojet.class.php';
+require_once DOL_DOCUMENT_ROOT.'/core/lib/functions.lib.php';
+
+$devToolScriptName =  'UrlReplace';
+
+// Initialize technical object to manage hooks of page. Note that conf->hooks_modules contains array of hook context
+$hookmanager->initHooks(array('devcommunitytools'.$devToolScriptName));
+
+// Load translation files required by the page
+
+$action = GETPOST('action', 'aZ09');
+$ref = GETPOST("ref");
+$priceimpact = GETPOST("priceimpact");
+$updateproductprices = GETPOST("updateproductprices");
+$update_status = GETPOST("update_status");
+$logManager = new devCommunityTools\LogManager();
+$form = new Form($db);
+$token = '';
+if(!function_exists('newToken')){
+	$token = $_SESSION['newtoken'];
+}
+else{
+	$token = newToken();
+}
+
+
+if($action == 'confirm_massive_update'){
+
+	if(!empty($updateproductprices)) {
+		$sql = "SELECT rowid FROM ".MAIN_DB_PREFIX."product WHERE ref LIKE '".$ref."%'";
```

Comment:
Warning : SQL injection possible

Reply:
Thank you ! I commited a fix, ref and temp_variation (even if not necessary ?) are escaped ;)